input/data/2017/2017ApJ...834....2A/tev-000049-sed-1.ecsv

```diff
@@ -2,8 +2,8 @@
 # ---
 # datatype:
 # - {name: e_ref, datatype: float32, unit: TeV}
-# - {name: e2dnde, datatype: float32, unit: TeV m-2 s-1}
-# - {name: e2dnde_err, datatype: float32, unit: TeV m-2 s-1}
+# - {name: dnde, datatype: float32, unit: TeV-1 m-2 s-1}
```

Comment:
Just to confirm: for `input/data/2017/2017ApJ...834....2A/tev-000049-sed-1.ecsv` was this change `e2dnde` -> `dnde` here intentional without touching the values?
The previous data entry was incorrect, and now it's correct?